AsyncDisplayKit/ASViewController.m

```diff
@@ -113,7 +149,9 @@ - (void)setAutomaticallyAdjustRangeModeBasedOnViewEvents:(BOOL)automaticallyAdju
 - (void)updateCurrentRangeModeWithModeIfPossible:(ASLayoutRangeMode)rangeMode
```

Comment:
How about updateCurrentRangeWithModeIfNeeded? It's pedantic, but updateCurrentRangeModeIfNeeded doesn't seem like it would take a parameter.